### PR TITLE
Fix for two issues in View.include

### PR DESCRIPTION
**Pull request by izapolsk**
1. View.include passes incorrect parent to included widgets #40 
2. Included widgets aren't available in inherited views #41

### CODE REVIEW

**Comment by mfalesni** [comment]
@izapolsk That behaviour is intentional, including is not designed to supplement complex logic inside the class being included.

**Comment by mfalesni** [comment]
@izapolsk Actually, it is described in readme, why it is done like that :)

**Comment by izapolsk** [comment]
@mfalesni , I apparently missed that in readme. Actually, I even didn't think it was intentionally done because everywhere else in widgetastic widget names are exposed in inherited classes.

**Comment by mfalesni** [comment]
I am okay with the inheritance thing, but the parent setting - I believe that needs to be done as a switch.

**Comment by izapolsk** [comment]
@mfalesni , awesome. regarding parent setting, it relevant to only included widgets. You pass there view's parent instead of current view. So, current view cannot be accessed by included widgets as a result.
Please give me more info what isn't ok.

**Comment by mfalesni** [comment]
@izapolsk Yep, the including view cannot be accesse currentlyd, there are two main reasons why I did that:
1) Widgets can specify a root locator, that can fence the lookup even though the included thing may be completely independent.
2) I don't think that the included widgets should necessary reach out and rely on specific widgets being in somewhere.

I am not a big fan of having the parent as the including widget, but I don't necessary oppose it, so I believe the `.include(View, use_parent=True)` would be a good use of that.

File: testing/test_basic_widgets.py [hunk 1]
```diff
@@ -597,6 +608,12 @@ def as_fill_value(self):
     assert not form.input1.fill(AFillable('a_test'))
     assert form.input1.read() == 'a_test'
 
+    # checking use_parent option
+    some_form = TestForm4(browser)
+    assert isinstance(some_form.input1.parent.parent, type(browser))
+    some_form2 = TestForm6(browser)
+    assert isinstance(some_form2.input1.parent.parent, TestForm6)
+
```
Comment (mfalesni) [inline testing/test_basic_widgets.py]: Please also add some widgets into the forms 4 and 6 and assert widget ordering.

**Comment by mfalesni** [comment]
@izapolsk I commented on the unit test, make sure the build passes and add coverage on the inherited widget including ordering. Could you also please modify the readme to include the mention of use_parent? Rephrase the article that there are two possibilities etc ...

**Comment by izapolsk** [comment]
@mfalesni, please check whether this is what you expected to see.

**Pull request merged**